datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -270,7 +312,9 @@ export default function DefaultPreviewCard({
         event.stopPropagation();
     };
 
-    const shouldShowRightColumn = (topUsers && topUsers.length > 0) || (owners && owners.length > 0);
+    const statusPillColor = status === 'SUCCESS' ? 'green' : 'red';
+    const shouldShowRightColumn =
```

Comment:
@asikowitz added additional condition to `shouldShowRightColumn` -- lmk if this makes sense